be/src/runtime/decimalv3.h

```diff
@@ -189,13 +189,19 @@ class DecimalV3Cast {
     }
 
     template <typename T>
-    static constexpr T float_overflow_indicator = std::numeric_limits<T>::lowest();
+    static constexpr T float_lower_overflow_indicator = std::numeric_limits<T>::max();
+    template <typename T>
+    static constexpr T float_upper_overflow_indicator = std::numeric_limits<T>::min();
+
     template <typename From, typename To>
     static inline bool from_float(FloatType<From> value, DecimalType<To> const& scale_factor,
                                   DecimalType<To>* dec_value) {
         *dec_value = static_cast<To>(scale_factor * static_cast<double>(value));
         if constexpr (is_decimal32<To> || is_decimal64<To>) {
-            return *dec_value == float_overflow_indicator<To>;
+            // Depending on the compiler implement, std::numeric_limits<T>::max() or std::numeric_limits<T>::max() both could be returned,
```

Comment:
Add some concise comment about different architecture.

Reply:
Detail about the difference of these arch is added.